runtime/src/accounts_index.rs

```diff
@@ -7,6 +7,8 @@ use std::{
 
 pub type SlotList<T> = Vec<(Slot, T)>;
 pub type SlotSlice<'s, T> = &'s [(Slot, T)];
+pub type AncestorList = HashMap<Slot, usize>;
```

Comment:
I've chosen this name to align with those preceding type aliases.

Reply:
Also, I've selected `???List` because I wanted to introduce some thin abstraction from the underlying types (`SlotList` from `Vec`; `AncestorList` from `HashMap`) for the case that we switch the container types in the future for implmentation changes.

I know `???List` is a bit ambiguous to `std::collections::LinkedList`, but `???Collection` or `???Container` is too long to type...

Anyway, this is not my strong opinion. :)